### PR TITLE
EOS-20050: Cluster: Factory Reset: Config

### PR DESCRIPTION
**Pull request by sradhanand-pati**

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: srv/components/provisioner/scripts/salt.sh [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    if [[ "$try" -gt "$max_tries" ]]; then
+        echo "ERROR: Minion $minion_id seems still not ready after $max_tries attempts." >> "${LOG_FILE}"
+        echo "ERROR: Cortx provisioner environment configuration failed" >> "${LOG_FILE}"
+        exit 1
+    fi
+    try=$(( try + 1 ))
+done
+echo "DEBUG: Salt configuration done successfully" >> "${LOG_FILE}"
+echo "Cortx provisioner environment configured successfully" >> "${LOG_FILE}"
+echo "Done" >> "${LOG_FILE}"
```
Comment (nitin-seagate) [inline srv/components/provisioner/scripts/salt.sh:90]: Add new line to file...

**Review by nitin-seagate — APPROVED** [review]
Verify with test results

File: srv/components/provisioner/scripts/salt.sh (hunk 1 above)
Comment (sradhanand-pati) [reply srv/components/provisioner/scripts/salt.sh:90]: Done

**Review by nitin-seagate — APPROVED** [review]

**Review by nitin-seagate — APPROVED** [review]

**Pull request merged**